### DISCUSSION TITLE
Maintenance notices

### DISCUSSION THREAD

**Discussion by shentino** [ChatTheatre/Community#6] [General]
This thread will be used for maintenance notices for gables.

Up first is that a kernel update was completed yesterday and the server will be rebooted at 6:00 am pacific on Friday, October 23, 2020

**Comment by noahgibbs** [comment]
Is this a kernellib update inside DGD, or a Linux kernel update?

**Comment by shentino** [comment]
OS kernel update, sorry :P.

As for the kernellib update on top of DGD that layer of code is pretty much frozen

**Comment by ChristopherA** [comment]
@shentino,

Can I get an idea on what other maintenance tasks, and on what schedule, you are doing for all the former Skotos games?

Thanks!

**Reply by shentino** [reply]
Every monday I check for package updates on debian and if any are found I upgrade.  If a reboot becomes needed after to complete the upgrade I send notice to the storyhost of the corresponding game about the reboot, to be completed on the friday of that week.

I also make myself available for technical consultation regarding DGD's inner workings or the lower layers of the code.

**Comment by shentino** [comment]
Kernel updated.
Pending reboot: Friday, December 11, 2020 at 6:00 am pacific

**Comment by shentino** [comment]
Kernel updated.
Reboot pending for Friday, February 19, 2021, at 6:00 am pacific

**Comment by shentino** [comment]
Kernel updated.

Reboot pending for Riday, September 10, 2021, at 6:00 am pacific

also sorry for falling behind, I kinda suck at staying disciplined and I had some drama to deal with IRL.

**Comment by shentino** [comment]
Kernel on gables updated today.

Reboot pending for Friday, October 15, 2021 at 6:30 am pacific time.

Comment:  Turns out I actually need some time after 6:00 am to actually finish waking up :P.

**Comment by shentino** [comment]
Kernel updated.

Reboot pending for Friday, July 8, 2022 at 7:00 am pacific time
